src/component/feature/view-edit-container.jsx

```diff
@@ -109,24 +109,24 @@ class EditFeatureToggleWrapper extends React.Component {
                         }
                         <AppsLinkList apps={seenApps} />
                     </Cell>
-                    <Cell col={3}>
-                            <div><strong>History</strong></div>
-                            <List style={{ textAlign: 'left' }}>
-                                {history.map(({ createdAt, type, createdBy }, i) =>
-                                    <ListItem twoLine key={i}>
-                                        <ListItemContent title={type} avatar={getIcon(type)} subtitle={createdAt}>
-                                            {createdBy}
-                                        </ListItemContent>
-                                    </ListItem>)}
-                            </List>
-                            <Link to={`/history/${featureToggleName}`}>
-                                See all events.
-                            </Link>
-                    </Cell>
                 </Grid>
             </div>
-            ) : (
+            ) : this.state.activeTab === 1 ? (
                 <EditFeatureToggle featureToggle={featureToggle} />
+            ) : (
+                <div>
+                    <List style={{ textAlign: 'left' }}>
+                        {history.map(({ createdAt, type, createdBy }, i) =>
+                            <ListItem twoLine key={i}>
+                                <ListItemContent title={type} avatar={getIcon(type)} subtitle={createdAt}>
+                                    {createdBy}
+                                </ListItemContent>
+                            </ListItem>)}
+                    </List>
+                    <Link to={`/history/${featureToggleName}`}>
+                        See all events.
+                    </Link>
+                </div>
```

Comment:
Nice to have it in its own tab, but should we then just have it the full History-table here instead of linking to "all events"?